integration-test/pkcs11/test-config/core_pkcs11_config.h

```diff
@@ -0,0 +1,176 @@
+/*
+ * FreeRTOS V1.1.4
```

Comment:
Please make the change in all files
```suggestion
 * AWS IoT Device SDK for Embedded C
```

Reply:
Ah right, thank you